### PR TITLE
LPS-135604 Updating correctly field affect by calculate rule

### PR DESCRIPTION
**Pull request by rijema**
Click [here](https://issues.liferay.com/browse/LPP-42083) to see the LPS

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorRuleHelper.java [hunk 1]
```diff
@@ -119,6 +140,21 @@ protected boolean containsAction(
...
+		String setStringPropertyAction = String.format(
+			"%s('%s', %s)", functionName, ddmFormFieldName, defaultValue);
+
+		List<String> actions = ddmFormRule.getActions();
+
+		Stream<String> stream = actions.stream();
+
+		return stream.anyMatch(
+			action -> Objects.equals(setStringPropertyAction, action));
+	}
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorRuleHelper.java:156]: @rijema, I've realized that this new method has the same logic from the existing one, basically you just need a different type for the parameter `defaultValue`, am I right?

If so, can you try to use the existing method `containsAction` and change the type of the parameter `defaultValue` to something more generic, like `Object`?

I feel that you can reuse the existing method, can you please give it a try? :slightly_smiling_face: 

Thanks

**Comment by natocesarrego** [comment]
Hi @rijema, I'm closing this pull for now due to inactivity. Once you get some time, please send a new pull. :slightly_smiling_face: 

Thanks.

**Comment by rijema** [comment]
Hi @natocesarrego, I'm sorry for not giving you an answer. I've found some struggles with this implementation on master and also found a regression bug that I'm working to solve. The CS also changed the steps of the ticket so it also impacted the previous fix.  I reopened an PTR but in the meanwhile I'm working to solve the regression bug. 

Regards

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorRuleHelper.java (hunk 1 above)
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorRuleHelper.java:156]: Changes made.

**Comment by rijema** [comment]
[LPS](https://issues.liferay.com/browse/LPP-42083)
[Bug that returned](https://issues.liferay.com/browse/LPS-110839)
[PTR](https://issues.liferay.com/browse/PTR-2547)

**Comment by rijema** [comment]
Hi @natocesarrego , I just sent a new one here https://github.com/liferay-forms/liferay-portal/pull/793

**Pull request closed without merging**